denizen_scripts/HeroCraft/mobs/suffixes/reaping.dsc

```diff
@@ -0,0 +1,9 @@
+custom_mob_suffix_reaping:
+  Type: world
+  debug: false
+  events:
+    after entity dies chance:50:
+        - define reaping <context.location.find_entities[*].within[10].filter_tag[<[filter_value].has_flag[reaping]>]>
```

Comment:
What applies reaping?

Reply:
oops i forgotted

Reply:
actually, what do you mean?

Reply:
@Ph4i1ur3

Reply:
Oh, I see! Let's make this more fun, shall we? Increase max health proportionately. heheh

Reply:
oh god